content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -225,3 +225,10 @@ One use case for selecting over labels is to constrain the set of nodes onto whi
 See the documentation on [node selection](/docs/concepts/configuration/assign-pod-node/) for more information.
 
 {{% /capture %}}
+
+{{% capture whatsnext %}}
+* Learn more about [labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
+* Learn more about [different ways to use labels](/docs/concepts/overview/working-with-objects/common-labels/#examples).
```

Comment:
It is good to have the above pointers, but the current text may be a little confusing. We are already on the page which focuses on the "label" concept, why are we redirected to some where else to "learn more about labels". The two link targets above are both about "common labels", or "recommended labels". So ... we may want to tweak the texts to avoid such confusion.

Reply:
@tengqm Sorry for the delay in implementing review comments. Made changes as per your suggestion. Please have a look at it.